core/market/src/testing/mock_node.rs

```diff
@@ -110,6 +110,7 @@ impl MarketsNetwork {
     /// It will be used to create directories and GSB binding points,
     /// to avoid potential name clashes.
     pub async fn new(test_name: Option<&str>) -> Self {
+        std::env::set_var("GSB_URL", "wrong url");
```

Comment:
Could you add a comment why is this needed?

Reply:
good catch! thx. Will do